it/uk/gov/hmrc/minorentityidentificationfrontend/stubs/StorageStub.scala

```diff
@@ -33,16 +33,35 @@ trait StorageStub extends WiremockMethods {
       status = status
     )
 
+  def stubStoreStoreTrustsKnownFacts(journeyId: String)(status: Int, body: JsValue = Json.obj()): Unit =
+    when(method = PUT, uri = s"/minor-entity-identification/journey/$journeyId/trustKnownFacts").thenReturn(
+      status = status,
+      body = body
+    )
+
+  def stubStoreIdentifiersMatch(journeyId: String)(status: Int, identifiersMatch: KnownFactsMatchingResult): Unit =
+    when(method = PUT, uri = s"/minor-entity-identification/journey/$journeyId/identifiersMatch").thenReturn(
+      status = status,
+      body = Json.obj("identifiersMatch" -> identifiersMatch)
+    )
+
+  def stubRetrieveIdentifiersMatch(journeyId: String)(status: Int, optIdentifiersMatchJsonValue: Option[String]): Unit =
+    when(method = GET, uri = s"/minor-entity-identification/journey/$journeyId/identifiersMatch").thenReturn(
+      status = status,
+      body = optIdentifiersMatchJsonValue match {
+        case Some(value) => Json.obj("identifiersMatch" -> value)
+        case None        => JsObject.empty
+      }
+    )
```

Comment:
Can we return a JsValue, similar to below instead of matching in this. The backend call that we are stubbing here doesn't return this so doesn't seem accurate to do it

Reply:
... I did not get it .... there is not possibility to return JsObject.empty??

Reply:
now it is clear ... I will fix it.